### PR TITLE
Removed font awesome icons from appearing on status messages

### PR DESCRIPTION
**Pull request by praneethpr**

### CODE REVIEW

**Comment by mickel1138** [comment]
To really remove them, the <i> tags in each template should be removed.

**Comment by praneethpr** [comment]
> To really remove them, the tags in each template should be removed.

Oh, I just checked the code. It looks like all font awesome icons were added in this epic

**Comment by mickel1138** [comment]
> > To really remove them, the tags in each template should be removed.
> 
> Oh, I just checked the code. It looks like all font awesome icons were added in this epic

You could almost revert back from develop.

File: src/Pelagos/Bundle/AppBundle/Resources/views/Account/AccountExists.html.twig [hunk 1]
```diff
@@ -3,7 +3,6 @@
 {% block body %}
     <div class="messages error alert alert-danger">
         <h2 class="element-invisible">Error message</h2>
-        <i class="fas fa-exclamation-circle"></i>
         The email address you have entered is already associated with a GRIIDC account. If you have forgotten your password or would like to create a new password please visit <a href="{{ path('pelagos_app_ui_account_passwordreset') }}" class="alert-link">password reset</a>. For any additional questions please contact <a href=
             "griidc@gomri.org" class="alert-link">griidc@gomri.org</a>.
```
Comment (mickel1138) [inline src/Pelagos/Bundle/AppBundle/Resources/views/Account/AccountExists.html.twig]: These alert link are bootstrap also, it wouldn't hurt it they stayed, but what's worse is that the link is wrong, it should have email: in front of it. So we can add it, or remove the A tag.

File: src/Pelagos/Bundle/AppBundle/Resources/views/Account/EmailNotFound.html.twig [hunk 2]
```diff
@@ -1,10 +1,9 @@
 {% extends "PelagosAppBundle:template:UI.html.twig" %}
 
 {% block body %}
-    <div class="messages error alert alert-warning">
+    <div class="messages error">
         <h2 class="element-invisible">Error message</h2>
-        <i class="fas fa-exclamation-circle"></i>
         The email address you have entered could not be found in the GRIIDC system. Please contact <a href=
-        "griidc@gomri.org" class="alert-link">griidc@gomri.org</a> for additional help.
+        "griidc@gomri.org">griidc@gomri.org</a> for additional help.
```
Comment (mickel1138) [inline src/Pelagos/Bundle/AppBundle/Resources/views/Account/EmailNotFound.html.twig]: I guess this is how it was in production too. Should we fixed this links, I think we should... Add email: before the e-mail addres.

Comment (mickel1138) [reply src/Pelagos/Bundle/AppBundle/Resources/views/Account/EmailNotFound.html.twig]: And off course it's "mailto:"...
https://www.iana.org/assignments/uri-schemes/uri-schemes.xhtml

File: src/Pelagos/Bundle/AppBundle/Resources/views/Account/AccountExists.html.twig [hunk 3]
```diff
@@ -1,10 +1,9 @@
...
 
 {% block body %}
-    <div class="messages error alert alert-danger">
+    <div class="messages error">
         <h2 class="element-invisible">Error message</h2>
-        <i class="fas fa-exclamation-circle"></i>
-        The email address you have entered is already associated with a GRIIDC account. If you have forgotten your password or would like to create a new password please visit <a href="{{ path('pelagos_app_ui_account_passwordreset') }}" class="alert-link">password reset</a>. For any additional questions please contact <a href=
-            "griidc@gomri.org" class="alert-link">griidc@gomri.org</a>.
+        The email address you have entered is already associated with a GRIIDC account. If you have forgotten your password or would like to create a new password please visit <a href="{{ path('pelagos_app_ui_account_passwordreset') }}">password reset</a>. For any additional questions please contact <a href=
+            "griidc@gomri.org">griidc@gomri.org</a>.
```
Comment (mickel1138) [inline src/Pelagos/Bundle/AppBundle/Resources/views/Account/AccountExists.html.twig]: mailto here too then...

File: src/Pelagos/Bundle/AppBundle/Resources/views/Account/NoAccount.html.twig [hunk 4]
```diff
@@ -1,10 +1,9 @@
...
 
 {% block body %}
-    <div class="messages error alert alert-danger">
+    <div class="messages error">
         <h2 class="element-invisible">Error message</h2>
-        <i class="fas fa-exclamation-circle"></i>
-        The email address you have entered is not associated with a GRIIDC account. If you would like to request an account, please visit <a href="{{ path('pelagos_app_ui_account_default') }}" class="alert-link">Account Request</a>. For any additional questions please contact <a href=
-            "griidc@gomri.org" class="alert-link">griidc@gomri.org</a>.
+        The email address you have entered is not associated with a GRIIDC account. If you would like to request an account, please visit <a href="{{ path('pelagos_app_ui_account_default') }}">Account Request</a>. For any additional questions please contact <a href=
+            "griidc@gomri.org">griidc@gomri.org</a>.
```
Comment (mickel1138) [inline src/Pelagos/Bundle/AppBundle/Resources/views/Account/NoAccount.html.twig]: This needs it too.

File: src/Pelagos/Bundle/AppBundle/Resources/views/template/ErrorMessage.html.twig [hunk 5]
```diff
@@ -5,9 +5,8 @@
 {% endblock %}
 
 {% block body %}
-    <div class="messages error alert alert-danger">
+    <div class="messages error">
         <h2 class="element-invisible">Error message</h2>
-        <i class="fas fa-exclamation-circle"></i>
         {{errormessage}}. For any additional help please contact <a href=
         "griidc@gomri.org">griidc@gomri.org</a>.
```
Comment (mickel1138) [inline src/Pelagos/Bundle/AppBundle/Resources/views/template/ErrorMessage.html.twig]: Needs mailto:

File: src/Pelagos/Bundle/AppBundle/Resources/views/template/InvalidToken.html.twig [hunk 6]
```diff
@@ -1,9 +1,8 @@
 {% extends "PelagosAppBundle:template:UI.html.twig" %}
 
 {% block body %}
-    <div class="messages error alert alert-danger">
+    <div class="messages error">
         <h2 class="element-invisible">Error message</h2>
-        <i class="fas fa-exclamation-circle"></i>
         The link you have selected is invalid. The link may have already been used to create an account. If you have forgotten your password please visit <a href="{{ path('pelagos_app_ui_account_passwordreset') }}">password reset</a>. For any additional help please contact <a href=
         "griidc@gomri.org">griidc@gomri.org</a>.
```
Comment (mickel1138) [inline src/Pelagos/Bundle/AppBundle/Resources/views/template/InvalidToken.html.twig]: and here

File: src/Pelagos/Bundle/AppBundle/Resources/views/template/NotLoggedIn.html.twig [hunk 7]
```diff
@@ -1,10 +1,9 @@
...
 
 {% block body %}
-    <div class="messages error alert alert-warning">
+    <div class="messages error">
         <h2 class="element-invisible">Error message</h2>
-        <i class="fas fa-exclamation-triangle"></i>
-        You have to be logged in to perform this action. If you have forgotten your password please visit <a href="{{ path('pelagos_app_ui_account_passwordreset') }}" class="alert-link">password reset</a>. For any additional help please contact <a href=
-        "griidc@gomri.org" class="alert-link">griidc@gomri.org</a>.
+        You have to be logged in to perform this action. If you have forgotten your password please visit <a href="{{ path('pelagos_app_ui_account_passwordreset') }}">password reset</a>. For any additional help please contact <a href=
+        "griidc@gomri.org">griidc@gomri.org</a>.
```
Comment (mickel1138) [inline src/Pelagos/Bundle/AppBundle/Resources/views/template/NotLoggedIn.html.twig]: Last one I think that needs mailto

**Pull request merged**